src/libs/actions/connections/ConnectToXero.ts

```diff
@@ -12,26 +12,18 @@ const getXeroSetupLink = (policyID: string) => {
     return commandURL + new URLSearchParams(params).toString();
 };
 
-/**
- * Fetches the category object from the xero.data.trackingCategories based on the category name.
```

Comment:
NAB why remove the comment?

Reply:
I've removed it since it was not relevant anymore as it's been replaced by `getTrackingCategories()`.

And as `getTrackingCategories()` is self-explaining, I felt like it was not necessary to add a comment for it.